examples/datasets/plot_iris_dataset.py

```diff
@@ -48,8 +55,20 @@
 plt.xticks(())
 plt.yticks(())
 
-# To getter a better understanding of interaction of the dimensions
-# plot the first three PCA dimensions
+# Each point in the scatter plot refers to one of the 150 iris flowers
```

Comment:
To show this text as a markdown cell, we need to add the `# %%` marker.

```suggestion
# %%
# Each point in the scatter plot refers to one of the 150 iris flowers
```

Reply:
Since I propose to add the legend, you can provide more description: the setosa type is easily identifiable since it has a short and large sepal. The remaining two types are easily identifiable using only the sepal measurement.